extensions/wikia/Email/Controller/WatchedPageController.class.php

```diff
@@ -67,7 +67,7 @@ protected function assertValidTitle() {
 	protected function getFooterMessages() {
 		$footerMessages = [
 			$this->getMessage( 'emailext-unfollow-text',
-				$this->title->getCanonicalUrl( 'action=unwatch' ),
+				$this->title->getCanonicalURL( 'action=unwatch' ),
```

Comment:
These 2 changes are just cosmetic clean-ups. I updated these calls to use the same case as the actual method declaration.